storage-node/src/bin/storage_node.rs

```diff
@@ -1,9 +1,12 @@
+use log::info;
+use storage_common::init_logger;
 use storage_node::server::storage_node_serve;
 
 // TODO: Add config here.
 
 #[tokio::main]
 async fn main() {
-    println!("starting storage node server...");
+    init_logger();
```

Comment:
I don't think we should set `is_test(true)` in the main function

Reply:
Then perhaps pass a boolean flag as a param to `init_logger()`? It seems ugly tho...

Reply:
Since it's the main function and will only be called once, it should be fine to directly write the init statement here rather than calling a function?